runtime/tools/ccfilter.c

```diff
@@ -291,8 +291,9 @@ int main( int argc, char *argv[] )
 			prefetch = 0;
 		      }
 		     else
-		      { strcat( Line, "\n" );
-			strcat( Line, Line2 );
+		      {
+			char	Line[2*LINELENGTH + 1];
+			char	Line2[LINELENGTH];
```

Comment:
This is totally wrong.
@micbou's suggestion is changing the definition of `Line` in the line 24, not here.